src/api/app/models/user.rb

```diff
@@ -983,6 +983,10 @@ def update_notifications(params)
   end
 
   private
+  # This method returns an array which contains all valid hash types.
+  def self.default_password_hash_types
```

Comment:
That method is already defined in line 185. Please add the code there.

btw. this is a class method. If there is a good reason to make it private we would have to use [private_class_method](http://ruby-doc.org/core-2.0.0/Module.html#method-i-private_class_method). But I doubt this is necessary here.